### ISSUE TITLE
Deeper dive reviewing Zipbooks & Open Collective

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/iced-coffee#347]
- [ ] Investigate the pros/cons of both platforms. We talked about wanting something that is free or freemium but provides structure in accounting. _A more simple alternative to these two would be using Google Sheets but it would not have as much structure._

Do they need to connect to a bank account for us to demo?

- [ ] Reach out Code for Miami about Open Collective use - @mateoclarke

**Comment by JMensch** [comment]
Some notes:

### Overview
- Zipbooks
Accounting software for the typical small business business (customers, invoices, expenses). Accepts CC's and paypal but nothing built in for donations. 

- Open Collective
Platform to automatically receive & manage donations, expenses, and to some extent an open source community. Built in transparency and other processes typical to popular (read: backed with donations) open source projects. Some overlap with meetups.com. 

### Facts
- Cost
  - Z: 1 bank account for free, CO: free.
- Cost for money in
  - Z: free, CO: 5% (10% for automated accounting + taxes)
- Cost for money out
  - Z: N/A (unavailable to pay out via app), OC: free.

### My Opinions
- Zipbooks looks like a comprehensive accounting tool, but provides little else. OC has heftier fees but provides a platform to automate donations and transparency. Additionally, OC's % takes are negligent for us right now.

Here's code for denver's open collective page: https://opencollective.com/codefordenver

**Comment by mateoclarke** [comment]
Tonight I spoke with Willy from Code for Denver about their Open Collective usage. Here are my notes...

TL;DR: Its worth trying, avoid the events features.

Last year, CfD was trying to get more funders to be less dependent on 1 single chunk grant for the past couple years. They had been successful with grants but it was always stressful and a pain to do. 

They found a sponsor in Waffle to donate. Waffle had an open collective account and asked CfD to donate via OC. 

They used the Events feature on OC, on top of just open source funding mechanism to try to get donations per number of attendee. $10 per person that shows up, up to a certain amount for each event. Tried doing ticketing through Open Collective, only lasted for a couple week, gave feedback to OC. They decided to go back to waffle and renegotiated the agreement to count attendees via Meetup and eventually just a flat recurring monthly donation.

Once you have funds with OC, the expense process is easy, give certain members to have the admin privilege to give the ok to reimburse expenses. Anyone can submit and upload a receipt. Low friction for expenses.

**Comment by JMensch** [comment]
That makes sense- thanks for taking the time. I propose we move forward with OC (#355). At minimum we can get our donations back up and running quickly (#340). We can re-evaluate OC after the 501c3 decision has been finalized. Thoughts @mateoclarke ?

**Comment by mateoclarke** [comment]
Yeah, I agree! You want to run with the account setup, or me?

**Comment by mateoclarke** [comment]
Yeah, I agree! You want to run with the account setup, or me? @JMensch

**Comment by JMensch** [comment]
I can handle it! Aiming to get something to show for Wednesday.
